Automattic/wp-calypso
# Component: Migrate Suggestions to TypeScript

#### Changes proposed in this Pull Request

* Migrate `Suggestions` and `Item` to TypeScript

#### Testing instructions



* Ensure build passes
* Go to `/devdocs/design` and search `Suggestions` and ensure the components continue to work as expected
